### PR TITLE
Adding a few ideas around scheduling

### PR DESCRIPTION
**Pull request by alexnm**
Added a few ideas to the scheduling part. Let me know what you think.

Awesome list overall! I'll try to see if we can contribute with more stuff from our recent experience

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,6 +103,11 @@ So many conferences got this wrong, and it's so simple. It shouldn't be anything
 * Consider having varying talk lengths. Having only 30 minute talks can feel monotonous - throw in some 15's or 20's to keep the rhythm dynamic.
 * Have many, long breaks. While the talks are interesting, the real magic at conferences happens in the conversations with people who are as passionate about the topic as you are. ReactRally is one of the conferences who does this really well with a 30 minute break after every 2 talks.
 * Schedule a big-name speaker to the final block of the day so people are more likely to stay for the whole day - if you're a new or less-known name, it sucks to speak at the end of the day to a half-empty audience.
+* Keep a timer somewhere in the front row where the speaker can clearly see it.
+* **Do not**, under any circumstances, interrupt the speaker while he's still talking. Allow your schedule to be a bit flexible. If you're 10 minutes behind, take 5 minutes from the next break or from the lunch break, but let the speakers finish their talks. It's all about respecting the work they put to be on stage.
```
Comment (lukeb-uk) [inline README.md]: I'm unsure about this one. The speaker overrunning is a huge sign of disrespect to the organisers and audience. I feel that while a certain buffer is acceptable, going 10 minutes over time on what was supposed to be a 30 minute talk is not.

Comment (kyleshevlin) [reply README.md]: If we're going to make this change, please make it gender neutral language. Do not assume "he's" talking.

Comment (alexnm) [reply README.md]: thanks! I missed that, did not had a last check before the commit

**Comment by lukeb-uk** [comment]
I do think the speaker should be stopped within a reasonable buffer of their time if they're over. They for sure should not be cut short because the conference is running behind.

Speaker into their 35th minute: cut short
Speaker in their 20th minute but the conference is behind: let them finish

**Comment by markdalgleish** [comment]
Keep in mind that at a multi track conference, speakers *definitely* shouldn't go over their allotted time or it stops attendees from being able to change rooms properly.

As a side note, timekeepers should be provided with the means to warn speakers if the more subtle signals aren't working, e.g. a bell to ring. I've been a timekeeper before, and one time I needed to warn the speaker they were going over but had no way of doing it—unless I wanted to walk up to the stage and start waving my hands, which I really didn't feel comfortable doing.

**Comment by jackblackCH** [comment]
A time controlled light in front of the stage that dimmes softly into a red light, the closer the time is over. That could be a good descreate help.

File: README.md [hunk 2]
```diff
@@ -103,6 +103,11 @@ So many conferences got this wrong, and it's so simple. It shouldn't be anything
 * Consider having varying talk lengths. Having only 30 minute talks can feel monotonous - throw in some 15's or 20's to keep the rhythm dynamic.
 * Have many, long breaks. While the talks are interesting, the real magic at conferences happens in the conversations with people who are as passionate about the topic as you are. ReactRally is one of the conferences who does this really well with a 30 minute break after every 2 talks.
 * Schedule a big-name speaker to the final block of the day so people are more likely to stay for the whole day - if you're a new or less-known name, it sucks to speak at the end of the day to a half-empty audience.
+* Keep a timer somewhere in the front row where the speaker can clearly see it.
+* **Do not**, under any circumstances, interrupt the speaker while the talk is in progress. Allow your schedule to be a bit flexible. If you're 10 minutes behind, take 5 minutes from the next break or from the lunch break, but let the speakers finish their talks. It's all about respecting the work they put to be on stage.
```
Comment (MoOx) [inline README.md]: -1 for this, it's hard to organize conferences already

**Pull request merged**